index.php

```diff
@@ -0,0 +1,29 @@
+<?php 
+require_once('src/CollectionCardMaker.php');
+$db = new PDO('mysql:host=db;dbname=boardgames', 'root', 'password');
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db ->prepare("SELECT `name`, `description`, `playercount`, `difficulty` FROM `boardgames`");
+$query->execute();
+$boardgames = $query->fetchAll();
+?> 
+
+<html lang="en-GB"> 
+	<head>
+		<link rel="stylesheet" type="text/css" href="src/style.css">
+		<title>The boardgame collection</title>
+	</head>
+	<body>
+        <header>
+            <h1>Boardgame Collection</h1>
+        </header>
+		<div class = "card-container">
+			<?php
+			foreach ($boardgames as $boardgame){
+				$collectionCardMaker = new CollectionCardMaker($boardgame['name'], $boardgame['playercount'], $boardgame['difficulty'], $boardgame['description']);
```

Comment:
in the middle of the HTML page, we should only have very minimal PHP code that blurts out a string. creating objects is too logicy for that. instead, you should create objects at the top of the code before the html